wiki-docs/docs/operate/deploy/staking/stake.md

```diff
@@ -0,0 +1,75 @@
+In this section, we'll walkthrough how to stake WMATIC on the associated rootchain.
+
+## i. Initial Staking on the Rootchain
+
+Each validator needs to perform initial staking on the rootchain `StakeManager` contract. This is done using the `polygon-edge polybft stake` command. **Note that this command is for testing purposes only.**
+
+<details>
+<summary>Flags ↓</summary>
+
+| Flag                          | Description                                        | Example                                  |
```

Comment:
This needs to be updated. You already did it correctly in the wiki-docs/docs/operate/deploy/genesis-validators.md, so I would say just take the values from there.